kustomize/README.md

```diff
@@ -75,6 +77,9 @@ Here is the list of the variations available as Kustomize components that you co
   - The default Online Boutique deployment uses the in-cluster `redis` database for storing the contents of its shopping cart. The Memorystore deployment variation overrides the default database with its own Memorystore (Redis) database. These changes directly affect `cartservice`.
 - [**Integrate with Spanner**](components/spanner)
   - The default Online Boutique deployment uses the in-cluster `redis` database for storing the contents of its shopping cart. The Spanner deployment variation overrides the default database with its own Spanner database. These changes directly affect `cartservice`.
+- [**Integrate with AlloyDB**](components/alloydb)
+ - The default Online Boutique deployment uses the in-cluster `redis` database for storing the contents of its shopping cart. The AlloyDB deployment variation overrides the default database with its own AlloyDB database.
```

Comment:
```suggestion
  - The default Online Boutique deployment uses the in-cluster `redis` database for storing the contents of its shopping cart. The AlloyDB deployment variation overrides the default database with its own AlloyDB database.
```
It looks like GitHub expects 2 spaces for sub-bullets (see [current rendering here](https://github.com/GoogleCloudPlatform/microservices-demo/blob/d4d4d6ef603d293d02e3e719d31443dabe78dd98/kustomize/README.md#deploy-online-boutique-variations-with-kustomize)).